packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNav.tsx

```diff
@@ -40,6 +66,7 @@ export const useTeachingPopoverCarouselNav_unstable = (
         tabIndex: 0,
         ...props,
         children: rootChildren,
+        onKeyDown: handleKeyDown,
```

Comment:
Why not use[ `useArrowNavigationGroup()`](https://react.fluentui.dev/?path=/docs/utilities-focus-management-usearrownavigationgroup--default) here instead of the key down listener?

Reply:
We still require this to be a navigation group based on the Accessibility design, and useArrowNavigation does not seem to play well with useFocusableGroup